aioxmpp/muc/self_ping.py

```diff
@@ -31,6 +32,10 @@
 import aioxmpp.utils
 
 
+def _apply_jitter(v, amplitude):
+    return v + (random.random() * 2 - 1) * amplitude
```

Comment:
Should this not be `v * (1 + amplitude * (2*random.random() - 1))` for 10% relative jitter?

Reply:
Fixedyfix.